quantum-api/test/languages/javascript/changelog.spec.js

```diff
@@ -177,7 +186,11 @@ describe('changelog', () => {
         .class('qm-changelog-javascript-header')
           .add(dom.create('span')
             .class('qm-changelog-javascript-function')
-            .add(header('function', functionBuilder)(quantum.select(selection), transformer))))
+            .add(header('function', functionBuilder)(quantum.select(selection), transformer)))
+          // TODO: Fix in #144
+          .add(dom.create('span')
+            .class('qm-changelog-javascript-arg')
+            .add(header('property', propertyHeaderBuilder)(quantum.select(child1), transformer))))
```

Comment:
This is the issue with the changelog transform resulting in #144 

If the children of a constructor/function/method are included in the header, they are added to the end of the header - regardless of whether the change is for the constructor itself or not.